career_development/ladder.md

```diff
@@ -13,102 +13,112 @@
-#### iterative	
--[ ] well composed commit narratives in commit flows 
-#### communication	
--[ ] can help pm cultivate an asana board. understands kanban. can teach juniors to pair program.
-#### respectful teammate	
-#### hard skills
+#### Simple 
+-[ ] Does not over optimize. Understands when code is "good-enough".
+#### Test-driven	
+-[ ] Makes use of stubbing and mocking.
+-[ ] Understands tradeoffs of hitting the db. 
+#### Iterative	
+-[ ] Well-composed commit narratives in commit flows. 
+#### Communication	
+-[ ] Can help PM cultivate an Asana board.
+-[ ] Understands kanban.
+-[ ] Can teach juniors to pair program.
+#### Respectful teammate
+-[ ] Communicate with PMs when goals are unattainable.
+-[ ] Understands the difficulties that the team is going through and helps how they can.	
+#### Hard skills
+-[ ] Demonstrates ability to succeed in a wide range of complex situations across multiple axes: e.g. scale, uncertainty, interconnectedness.
 
 ### Level 5
-#### simple 	
-#### test-driven	
-#### iterative	
--[ ] can guide an ipm and team reflection
-#### communication	
-#### respectful teammate	
-#### hard skills
+#### Simple 	
+-[ ] Extracts complex logic into small easy-to-follow classes/components.
+#### Test-driven
+-[ ]	
+#### Iterative	
+-[ ] Can guide an IPM and team reflection.
+#### Communication	
+-[ ] Aligns team on expectations.
+-[ ] Identifies miscommunications and solves them before they become a problem.
+#### Respectful teammate
+-[ ] Help dissolve tension when arises.
+-[ ] Does not take "all" responsibilities. They let their teammates take responsibilities they care about.	
+#### Hard skills
+-[ ] Breaks down complex projects into simple systems that can be effectively built and maintained by less experienced engineers.
 
 ### Level 6
-#### simple 	
-#### test-driven	
-#### iterative	
-#### communication	
-#### respectful teammate
--[ ] respects the right of junior team members to own parts of the code, can effectively run and explain 360 reviews	
-#### hard skills
-
-### Level 7
-#### simple 	
-#### test-driven	
-#### iterative	
-#### communication	
-#### respectful teammate	
-#### hard skills
-
-### Level 8
-#### simple 	
--[ ] has a clear understanding of long term maintenance costs and tradeoffs related to simplifying code
-#### test-driven	
-#### iterative	
-#### communication
--[ ] works to define tasks in asana and can mentor juniors in defining them. can collaborate with client about estimates and prioritization.	
-#### respectful teammate	
-#### hard skills
-
-### Level 9
-#### simple 	
-#### test-driven	
-#### iterative	
-#### communication	
-#### respectful teammate	
-#### hard skills
\ No newline at end of file
+#### Simple 	
+-[ ] Has a clear understanding of long term maintenance costs and tradeoffs related to simplifying code
+#### Test-driven	
+-[ ] Understands the cost of maintaining tests. Writes cost-effective tests.
```

Comment:
```suggestion
-[ ] Understands the cost of maintaining tests.
-[ ] Writes cost-effective tests.
```

Reply:
The reason why they are in the same line is that I believe they are related.
I'm OK with having them separately, though.